jb/testing/check_tde_result_close_enough.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef jb_testing_check_tde_result_close_enough_hpp
+#define jb_testing_check_tde_result_close_enough_hpp
+
+#include <jb/fftw/tde_result.hpp>
+#include <jb/testing/check_complex_close_enough.hpp>
+
+#include <boost/test/unit_test.hpp>
+
+namespace jb {
+namespace testing {
+
+/**
+ * Compare tde_result value within some tolerance.
+ *
+ * @return true if the numbers are within tolerance.
+ *
+ * @param actual the first tde_result to compare
+ * @param expected the second tde_result to compare
+ * @param tol expressed in:
+ * - "integer" if actual value type is integer
+ * - "number of epsilons" otherwise
+ *
+ * @tparam T the type of tde_result(s) to compare
+ */
+template <typename T>
```

Comment:
I think this is the third version of `close_enough` that basically amounts to iterating over a collection and calling `close_enough()` for each element: refactoring time???